cmd/minishift/cmd/registration/util.go

```diff
@@ -55,7 +55,7 @@ func RegisterHost(libMachineClient *libmachine.Client) {
 	}
 }
 
-func UnregisterHost(api libmachine.API, allowToSkipUnregistration bool) {
+func UnregisterHost(api libmachine.API, allowToSkipUnregistration bool, force bool) {
```

Comment:
can a full description be used for the variable, pretty much like `allowToSkipUnregistration`? eg. `forceDeletion` ?